python-app/main.py

```diff
@@ -62,18 +68,48 @@ def create_post():
     x = mycollections.insert_one(data)
 
     return "Success"
-    
+
+
 @app.route('/requests/delete', methods=["DELETE"])
-@cross_origin(origin='*',headers=['Content-Type','Authorization'])
+@cross_origin(origin='*', headers=['Content-Type', 'Authorization'])
 def delete_post():
     req = request.json
     # username = req["username"]
     course = req["course"]
     skill = req["skill"]
     msg = req["message"]
     tag = req["tag"]
-    
+
+
+@app.route('/api/profile', methods=["GET"])
+@cross_origin(origin='*', headers=['Content-Type', 'application/json'])
+def getProfileFromMongo(database="STUDYBUDDY", collection="user_details"):
+    mongoClient = pymongo.MongoClient(
+        "mongodb+srv://admin:admin@cluster0-jacon.gcp.mongodb.net/test?retryWrites=true&w=majority")
+    profiles = [x for x in mongoClient[database][collection].find()]
+    print("pulled {} profiles from MongoDB, total size: {} bytes".format(
+        len(profiles), str(sys.getsizeof(profiles))))
+    print(dumps(profiles[0]))
+    return dumps(profiles[0])
+
+
+@app.route('/api/profile', methods=["POST"])
+@cross_origin(origin='*', headers=['Content-Type', 'application/json'])
+def edit_profile():
+    req = request.json
+    data = {}
+    myclient = pymongo.MongoClient(
+        "mongodb+srv://admin:admin@cluster0-jacon.gcp.mongodb.net/test?retryWrites=true&w=majority")
+    mydb = myclient["STUDYBUDDY"]
+    mycollections = mydb["user_details"]
+    data["name"] = req["name"]
+    data["skills"] = req["skills"]
+    data["courses"] = req["courses"]
+    data["department"] = req["department"]
```

Comment:
where is the part to store the profile image?
You'll need to store the image on the server and persist the path in Mongo collection

Reply:
Image will be covered in later stories